app/src/ui/changes/no-changes.tsx

```diff
@@ -219,15 +220,14 @@ export class NoChanges extends React.Component<
     return (
       <>
         {parentMenusText} menu or{' '}
-        {this.renderDiscoverabilityKeyboardShortcut(menuItem)}
+        <KeyboardShortcut
+          darwinKeys={this.getMenuShortcut(menuItem)}
+          keys={this.getMenuShortcut(menuItem)}
+        />
       </>
     )
   }
 
-  private renderDiscoverabilityKeyboardShortcut(menuItem: IMenuItemInfo) {
-    return menuItem.acceleratorKeys.map((k, i) => <kbd key={k + i}>{k}</kbd>)
-  }
-
```

Comment:
I think it'd be less changes and more reused code if you kept this function around with this implementation:
```tsx
  private renderDiscoverabilityKeyboardShortcut(menuItem: IMenuItemInfo) {
    return <KeyboardShortcut
             darwinKeys={menuItemInfo.acceleratorKeys}
             keys={menuItemInfo.acceleratorKeys}
           />
  }
```

Then you can get rid of `getMenuShortcut` and revert the rest of changes in this file.